index.d.ts

```diff
@@ -16,10 +16,11 @@ interface TwimlConstructor<T> {
 declare function twilio(
   accountSid?: string,
   authToken?: string,
-  opts?: TwilioClient.TwilioClientOptions
-): TwilioClient;
+  opts?: _TwilioClient.TwilioClientOptions
+): _TwilioClient;
 
 declare namespace twilio {
+  export import TwilioClient = _TwilioClient;
```

Comment:
In `lib/index.js` this is being exported as `Twilio` and not `TwilioClient`. Could you update this please?

Reply:
Updated. Also reverted the '_' prefixing since renaming to Twilio removes the conflict with the local TwilioClient